tests/test_wcon.py

```diff
@@ -69,43 +25,152 @@ def test_wcon2df():
         assert col == col.upper()
 
 
-def test_str2df():
-    """Test dataframe extraction from strings"""
+def test_wconhist():
+    """Test WCONHIST parsing and column names"""
     wconstr = """
 WCONHIST
   'FOO' 0 1 /
  /
 """
     deck = EclFiles.str2deck(wconstr)
-    wcondf = wcon.df(deck)
-    assert len(wcondf) == 1
+    wconhist_df = wcon.df(deck)
+    pd.testing.assert_frame_equal(
+        wconhist_df,
+        pd.DataFrame(
+            [
+                {
+                    "WELL": "FOO",
+                    "STATUS": "0",
+                    "CMODE": "1",
+                    "ORAT": 0,
```

Comment:
Er 0 og None det samme her? Ser fra WSEGVALV testen med max blank at jeg tester mot None og ikke 0 for elementer som ikke er satt i tekststrengen.

Reply:
`0` and `None` are different Python objects, but sometimes act similarly. The statement

```
if not something:
    do_something()
```
will f.ex treat `something=0` and `something=None` identically.

In the dataframe, `0` and `None`means different things, and should be handled differently.